app/views/warehouse_reports/re_entry/index.haml

```diff
@@ -4,27 +4,24 @@
 = render '/warehouse_reports/breadcrumbs'
 
 = render 'filter'
-= render 'warehouse_reports/limited_by_project_note'
-
 - if @enrollments.any?
-
-  .row
-    .col.text-right
-      = link_to({format: :xlsx}.merge(link_params),{class: 'btn btn-secondary'}) do
-        Download Excel
-  .row
-    .col-sm-12
+  .d-flex
+    .mr-2
       %p
-        Clients re-entered homelessness between
+        Clients re-entered homelessness (ES, SO, SH, or TH) between
         = @filter.start
         and
         = @filter.end
         after a break of 60 days or more.
+        %br
+        This report was built to provide supporting details to the "Entry and Re-Entry" section of the Population Dashboards.
```

Comment:
Either move this up to the top of the `%p`, or move it down in the other two for consistency? I like it better here, if it makes a difference :-)